native/docs/manual-builds.md

```diff
@@ -29,24 +29,36 @@ found [here](build-configs.md).**
 - Install and setup XCode.
 - [Install and setup Fastlane](../../docs/cicd.md#fastlane-setup).
 - Read about [Apple Certificates](apple-certifcates.md).
+- To set up the certificates: have access to the `app-credentials` repo, and the Passbolt folder Integreat App
+  Publishing
 
 #### Certificates Setup
 
-- Prepare the necessary environment variables as
-  specified [here](../../docs/cicd.md#environment-variables-and-dependencies).
+- If none of your devices are logged in to the Apple account of the app team, warn someone with a logged in device
+  that you will need a code in a few minutes. That could be e.g. Steffi, Andy, or Leandra.
 
 - Install the certificates locally:
 
 ```bash
-cd ios && bundle exec fastlane certificates
+cd ios && fastlane match development (or whichever environment you need)
 ```
 
+- If you get an error that you can't clone the repo `app-credentials`, request access.
+
+- The password that you need to fill in twice is saved in Passbolt as Digitalfabrik Fastlane Match.
+
+- The Apple account that you need to sign in to is the one of the app team, the password is in Passbolt. This is also
+  the step where you will need the second factor.
+
+- You will need to add all the app_identifier you want to work with, in case of Integreat and its whitelabels, that would
+  be `de.integreat-app,app.aschaffenburg,de.malteapp.` If you fill in an invalid one, you get a list with the available ones.
+
 ##### Trouble Shooting
 
 - [Installing certificates hangs on `Cloning remote git repo...`](troubleshooting.md#bundle-exec-fastlane-certificates-hangs-on-cloning-remote-git-repo)
 - Secrets containing `'`: Exporting as environment variable is possible as
   follows: `export SECRET='<prefix>'"'"'<suffix>'`.
-- 2FA prompt: Message @Max or @Steffen.
+- 2FA prompt: Message Steffi, Andy, or Leandra.
```

Comment:
This info should rather be in passbolt where the passwort is, than in the repostitory.

Reply:
I'd add it in both places :)

Reply:
then we need to keep it updated in both places

Reply:
True